### DISCUSSION TITLE
聊聊沙井万丰海岸城买房打新经历 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#31] [Announcements]
# 聊聊沙井万丰海岸城买房打新经历 - forecho's Blog

引言
前几个月开始考虑在深圳买房，先看的是二手房，然后才知道还有买新房「打新」潮这个事情。二手房因为政府不好限价，新楼盘之前都有备案价，开盘的时候不能超过备案价的 15%，所以有一种现象就是新楼盘的价格会比周边二手房单价便宜几万的情况，其中最著名的事件就是「深圳润四网红盘」，买到就是赚到:

一手房和二手房之间有超过 5 万元/平方米的价差。以润四最小面积 100 平方米计算，买到一套房有 500 万元的价差空间。

深圳有钱人太多了。
言归正传，算了一下我和兔子在深圳两人截止到这个月一共交了 171 个月的社保了，还是有点优势的，不去「打新」不就浪费了？然后我们就开始去了解新楼盘，那已经是去年 11 月份了，深圳光明和沙井都涨了一波了 😭，然后我们了解到的最新网红楼盘就是「万丰海岸城」和「宏发万悦山」。只所以称为网红盘证明他们各方面都不错。

https://blog.forecho.com/about-coast-city.html

**Comment by fengzhao** [comment]
前几天海岸城业主维权，现在回过头看，你当时打新没中真是幸运啊

**Reply by forecho** [reply]
连海岸城都要维权？在中国买房子真难啊
